front/src/utils/react-id.ts

```diff
@@ -0,0 +1,10 @@
+import { v4 as uuidV4 } from 'uuid';
+
+/**
+ * Generate a unique id for react components
+ * @returns a unique id
+ */
+export default function nextId() {
+  // return `id-${uuidV4()}`;
+  return uuidV4();
```

Comment:
I think we can use `uuidV4()` directly in places where it makes sense, rather than introduce a wrapper function. (Note, the name "react-id" is slightly confusing, since these IDs aren't used in anything React-related.)